skyportal/handlers/api/sources_confirmed_in_gcn.py

```diff
@@ -684,14 +685,16 @@ async def post(self, dateobs):
                     loop = asyncio.new_event_loop()
                     asyncio.set_event_loop(loop)
 
+                request_body = {
+                    'obj_ids': [obj.obj_id for obj in sources_in_gcn],
+                    'tnsrobot_id': tnsrobot.id,
+                    'user_id': self.associated_user_object.id,
+                    'reporters': reporters,
+                }
+
                 IOLoop.current().run_in_executor(
                     None,
-                    lambda: tns_submission(
-                        [obj.obj_id for obj in sources_in_gcn],
-                        tnsrobot.id,
-                        self.associated_user_object.id,
-                        reporters=reporters,
-                    ),
+                    lambda: post_tns(request_body, timeout=30),
```

Comment:
Given that we have a post_tns method, I would create the request body there and just pass it objs, tnsrobot_id, user_id, and reporters. Just to avoid that recurring bit of code.

Reply:
Great idea.